pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -460,6 +460,29 @@ func getConsoleTemplate(cr *api.KieApp) api.ConsoleTemplate {
 		if cr.Status.Applied.Objects.Console.Cors != nil {
 			template.Cors = *cr.Status.Applied.Objects.Console.Cors
 		}
+
+		// Console StartupStrategy
+		if cr.Status.Applied.CommonConfig.StartupStrategy != nil {
+
+			if cr.Status.Applied.CommonConfig.StartupStrategy.StrategyName == api.OpenshiftStartupStrategy {
+
+				template.StartupStrategy.StrategyName = cr.Status.Applied.CommonConfig.StartupStrategy.StrategyName
+
+				if cr.Status.Applied.CommonConfig.StartupStrategy.ControllerTemplateCacheTTL == nil {
+					template.StartupStrategy.ControllerTemplateCacheTTL = Pint(5000)
+				} else {
+					template.StartupStrategy.ControllerTemplateCacheTTL = cr.Status.Applied.CommonConfig.StartupStrategy.ControllerTemplateCacheTTL
+				}
+			} else {
+
+				template.StartupStrategy.StrategyName = api.ControllerStartupStrategy
+			}
+
```

Comment:
Here I see some problem in using default strategy. When is no StartupStrategy set, then is used OSSS by default, but when there is a typo in strategy name (can happen when user writing KieApp.yaml manually) then is used Controller strategy.
Also we can remove duplicated code for setting up `template.StartupStrategy.StrategyName`.

I would suggest, to edit this part.
Check that applied strategy name is one of allowed values
- If yes, set it to template `template.StartupStrategy.StrategyName = cr.Status.Applied.CommonConfig.StartupStrategy.StrategyName`
- If not set up the default strategy (OSSS) - in same way as if would no strategy set up

Reply:
ok